Makefile.org

```diff
@@ -418,12 +418,22 @@ clean:	libclean
 	rm -f $(LIBS)
 	rm -f openssl.pc libssl.pc libcrypto.pc
 	rm -f speed.* .pure
+	$(RM) test/newkey.pem test/testkey.pem test/testreq.pem
+	-$(RM) `find . -name '*.o' -a \! -path "./.git/*"`
```

Comment:
You mean make dclean ?
The problem with make dclean is that after it has executed, git sees
lots of changes and you cannot change to another branch any more.